jwql/website/apps/jwql/templates/archive.html

```diff
@@ -2,73 +2,123 @@
 
 {% block preamble %}
 
-	<title>Archived {{ inst }} Images - JWQL</title>
+    <!-- Custom styles and scripts for this template -->
+    <style type="text/css">
+        div#proposal-array { display: none; }
+    </style>
+
+    <script type="text/javascript">
+    /**
+     * Updates various compnents on the archive page
+     * @param {String} inst - The instrument of interest (e.g. "FGS")
+     * @param {String} base_url - The base URL for gathering data from the AJAX view.
+     */
+    function update_archive_page(inst, base_url) {
```

Comment:
@laurenmarietta Ah yes, sorry I should have mentioned this before --

I am finding that any javascript function that references `src="{{static()}}"` in some way (e.g. `static("thumbnails/")` does not work when they live in `jwql.js`.  This seems to indicate that `static()` points to a path that is relative to the `website/apps/jwql/static/` directory instead of `website/apps/jwql/static/js/` directory.  Perhaps this has something to do with how we define the static paths in our `settings.py` file.  For now, I just kept things within the HTML template so that it works, but I do think we should figure out a better way in a separate PR, if you agree.